### PR TITLE
Check for existing message without id when adding messages

### PR DESCRIPTION
**Pull request by lemieux**
A small bug caused sent messages to appear twice for a fraction of a second when it was received by faye.

This change will ensure the new message won't get added to the conversation.

@mspensieri @Mario54 @dannytranlx

### CODE REVIEW

**Comment by Mario54** [comment]
:+1:

File: src/js/reducers/conversation-reducer.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ const sortMessages = (messages) => messages.sort((a, b) => {
 });
 
 const addMessage = (messages, message) => {
-    const existingMessage = messages.find((m) => m._id === message._id);
+    let existingMessage = messages.find((m) => m._id === message._id);
+
+    // let's try to match against recently sent messages instead
+    if (!existingMessage) {
```
Comment (mspensieri) [inline src/js/reducers/conversation-reducer.js]: Could check for `role === 'appUser'` here as a small optimization. App maker messages that come in through faye shouldn't suffer from this bug

**Comment by mspensieri** [comment]
:+1:

File: src/js/reducers/conversation-reducer.js (hunk 1 above)
Comment (lemieux) [inline src/js/reducers/conversation-reducer.js]: true, I'll add that

**Pull request merged**